lib/exercises.js

```diff
@@ -11,6 +11,13 @@ export const exercises = [
       "Keep your elbows at a 45-degree angle from your body.",
       "Push back up to the starting position.",
     ],
+    intro:
+      "A classic bodyweight exercise that targets the chest, shoulders, and triceps.",
+    benefits: [
+      "Builds upper body strength",
+      "Improves core stability",
+      "Enhances functional fitness",
+    ],
   },
   {
     id: "2",
```

Comment:
Id uses strings "1", but you could reconsider using unique integers for id purposes.